### PR TITLE
list command for exercise

### LINKED ISSUE #6: As a user, I can list down all the exercises for the day
so that I can monitor the calories burned accurately

### PR DESCRIPTION
**Pull request by Nauman-S**
implemented list feature for v1.2

### CODE REVIEW

**Comment by aaronlwm** [comment]
Merge conflicts to resolve

**Comment by Nauman-S** [comment]
well done!

**Comment by Liu-2001** [comment]
LGTM.

**Comment by Nahoyhp** [comment]
Feel you bro. The amount of checkstyle error.

File: src/main/java/seedu/address/model/person/Calorie.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package seedu.address.model.person;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/Calorie.java]: Present in exercise

**Comment by Royxuzeng** [comment]
The code looks good.

File: src/main/java/seedu/address/model/person/Date.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/Date.java]: Present in exercise

File: src/main/java/seedu/address/model/person/Description.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/Description.java]: Present in exercise

File: src/main/java/seedu/address/model/person/Exercise.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
+package seedu.address.model.person;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/Exercise.java]: Present in exercise

File: src/main/java/seedu/address/model/person/exceptions/DuplicateExerciseException.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package seedu.address.model.person.exceptions;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/exceptions/DuplicateExerciseException.java]: Present in exercise

File: src/main/java/seedu/address/model/person/exceptions/ExerciseNotFoundException.java [hunk 6]
```diff
@@ -0,0 +1,3 @@
+package seedu.address.model.person.exceptions;
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/person/exceptions/ExerciseNotFoundException.java]: Present in exercise

**Comment by aaronlwm** [comment]
Great job

**Comment by Royxuzeng** [comment]
Fixes #6

**Pull request merged**